code/game/machinery/doors/airlock.dm

```diff
@@ -1272,15 +1272,15 @@
 
```

Comment:
This can be optimized to do

if(welded && locked)

if(welded)

if(locked)

not a request, but a thought

Reply:
no reason